packages/shared/src/components/auth/AuthModal.tsx

```diff
@@ -31,7 +31,11 @@ export default function AuthModal({
 }: AuthModalProps): ReactElement {
   const [showOptionsOnly, setShowOptionsOnly] = useState(false);
   const { authVersion } = useContext(FeaturesContext);
-  const { closeLogin } = useContext(AuthContext);
+  const { closeLogin, logout } = useContext(AuthContext);
+  const closeAndLogout = (e) => {
```

Comment:
I think `closeLogin` should have this functionality of closing the modal and logging out by default so there will be no way to close it without logging out

Reply:
Ah it's not always that you are already logged in, but we can default to it if you think it makes more sense.

Reply:
You can add a check inside the hook. But it's more fail-safe this way

Reply:
Ah no wait, but closeLogin is also called to simply close the modal, and that should not log out the person.
It's particular when they discard the changes only.

Reply:
Hmm I see